game/kernel/kmachine.cpp

```diff
@@ -172,22 +175,28 @@ void InitIOP() {
 
   if (!reboot) {
     // reboot with development IOP kernel
-    printf("Rebooting IOP...\n");
```

Comment:
these were in the game but i think are kinda useless so let's leave them as debug.